test.js

```diff
@@ -9,6 +10,10 @@ describe('sudo mode', function () {
 			return 0;
 		};
 
+		fs.statSync = function () {
```

Comment:
@twogood I think @kevva wants you to use a `sinon.spy()` instead as it won't completely bork up subsequential use of the function.